### PR TITLE
Move to Github actions

### LINKED ISSUE #88: Github actions
I think we can remove `travis` and move to Github actions.

Do we still need `scrutinzer`? Does this really add value?
We could move this to a static analyzer like `phpstan` or `psalm` (and run with a GH action).

What do you think @craigpaul ?

### PR DESCRIPTION
**Pull request by mvdnbrk**
Resolves #88

### CODE REVIEW

**Comment by mvdnbrk** [comment]
Think this is all good now.

Only thing I had to change was to bump guzzle to `^6.3` to make all tests passing.

@craigpaul Please use the `Squash and Merge` option, so we get a nice clean commit history 😉

**Comment by craigpaul** [comment]
Looks pretty good to me. One quick question though. How do we record/report code coverage? Should we set up an integration with something like codecov.io (after merging this)?

File: README.md [hunk 1]
```diff
@@ -1,11 +1,9 @@
 <p align="center"><a href="https://postmarkapp.com" target="_blank"><img src="https://postmarkapp.com/images/logo.svg" alt="Postmark" width="240" height="40"></a>
-    
+
 # Laravel Postmark
 
 [![Latest Version on Packagist][ico-version]][link-packagist]
 [![Software License][ico-license]](LICENSE.md)
-[![Build Status][ico-travis]][link-travis]
-[![Coverage Status][ico-scrutinizer]][link-scrutinizer]
 [![Quality Score][ico-code-quality]][link-code-quality]
```
Comment (craigpaul) [inline README.md]: I think we need to remove this

**Comment by mvdnbrk** [comment]
Don't know if we would really need that. 
I am always writing tests to make sure the code works as aspected.

If not a bug will pop up which should be fixed and tested to avoid problems in the future.

File: README.md (hunk 1 above)
Comment (mvdnbrk) [reply README.md]: Missed that one, thanks.

**Pull request merged**